packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -154,14 +153,8 @@ const VisualizationProvider: FC<Props> = ({
     invalidateCache,
 }) => {
     const itemsMap = useMemo(() => {
-        if (explore && resultsData)
-            return getItemMap(
-                explore,
-                resultsData.metricQuery.additionalMetrics,
-                resultsData.metricQuery.tableCalculations,
-                resultsData.metricQuery.customDimensions,
-            );
-    }, [explore, resultsData]);
+        return resultsData?.fields;
```

Comment:
This is where we stop using all the fields from the explore and only use the fields provided by the BE.